icon-registry/icons-static/document-square-minus_x12.svg

```diff
@@ -0,0 +1,4 @@
+<svg viewBox="0 0 12 12" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
@emilmilanov @tbiethman Do you know where this icon came from? I'm second guessing the name because it doesn't really look like a "document", but I can't find it in the Figma DS either to verify what the name of this icon is there.

Reply:
I didn't see an explicit name in [Figma](https://www.figma.com/design/zVWa3fJtO0EWZ5FEhqtCRL/Branch-Review%2C-v1.1---%40latest?node-id=3072-206228&t=uYjIG5f1CwjhBdVJ-0) so I named it similarly to the existing square added icon, `document-added-square-plus`. I might have put the cart before the horse if it's not in the Figma DS at all though.

Reply:
Based on Emil's feedback I'll update this to `modified_square_deleted` and add the additional sizes that are DS for this.